dstep/translator/CodeBlock.d

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Mar 11, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+module dstep.translator.CodeBlock;
+
+import std.array;
+import std.string;
+
+enum EndlHint
+{
+    Empty, // does nothing
+    Group,
+    SingleLine,
+    MultiLine, // indent children
+    SubscopeWeak, // put braces if children takes more than single line
+    SubscopeStrong, // always put braces
```

Comment:
Since you already have comments on most members here, why not make it proper ddoc comments and put it on all members.